pages/index.vue

```diff
@@ -30,8 +30,8 @@
       <section class="recent-jobs">
         <h2 class="section-title">Últimas vagas</h2>
         <ul class="job-list">
-          <li v-for="i in [1, 2, 3, 4, 5, 6]" :key="i" class="job-card-item">
-            <JobCard />
+          <li v-for="job in jobs" :key="job.id" class="job-card-item">
```

Comment:
Show to the user a spinner while the request is loading

Reply:
done!